cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,93 @@
+/* eslint-disable cypress/unsafe-to-chain-command */
+/* eslint-disable arrow-parens */
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
 
+  beforeEach(() => {
+    cy.visit('/');
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
+  });
+
+  it('should have pagination', () => {
+    // pagination
+    cy.get('.-pageInfo').should('contain', 'Page');
+    cy.get('.-pageJump > input').type('Number');
+    cy.get('.-totalPages').should('contain', '1');
+    cy.get('.-btn').should('contain', 'Previous');
+    cy.get('.-btn').should('contain', 'Next');
+  });
+
+  it('should have rows count selection', () => {
```

Comment:
its not enough to check values. Try to select some values and chech that table has the same number of rows that you have selected

Reply:
Done